docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -425,7 +429,7 @@ code: |
       #is_reserved_name = 
       if is_reserved_label(pdf_field_name) or pdf_field_name in people_variables.true_values():
         new_field = built_in_fields_used.appendObject()
-      elif pdf_field_tuple[4] == '/Sig':
+      elif len(pdf_field_tuple) > 4 and pdf_field_tuple[4] == '/Sig':
```

Comment:
Good catch, but when does this ever happen? I've never been able to find a good description of when these tuples aren't 5-ples.

Reply:
It can happen with some kind of fields--but it's not really needed here. It was part of my troubleshooting, but I'm OK with leaving it in.